QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java

```diff
@@ -37,14 +37,22 @@ protected void autonomousMain() throws AutonomousModeOverException, InterruptedE
 
             waitUntil(Elevator.atBottom);
 
-            Rollers.closed.set(true);
-            Rollers.direction.set(true);
-            Rollers.running.set(true);
-
-            waitUntil(BooleanMixing.invert(AutoLoader.crateInPosition));
-            waitForTime(timeout);
-
-            Rollers.running.set(false);
+            boolean r = Rollers.running.get();
+            boolean d = Rollers.direction.get();
+            boolean o = Rollers.closed.get();
```

Comment:
It's probably best to have more descriptive names for these. "running," "direction," and "open" would be fine.